packages/ag-charts-enterprise/src/setup.ts

```diff
@@ -80,9 +80,6 @@ export function setupEnterpriseModules() {
     );
 
     _ModuleSupport.enterpriseModule.isEnterprise = true;
-    _ModuleSupport.enterpriseModule.licenseManager = (options: AgChartOptions) =>
-        new LicenseManager(
-            options.container?.ownerDocument ?? (typeof document === 'undefined' ? undefined : document)
-        );
+    _ModuleSupport.enterpriseModule.licenseManager = new LicenseManager();
```

Comment:
As per my other comment, I would prefer to keep the `document` resolution out of `LicenseManager` for now?